mcp_server/tests/unit/test_gitlab_tools.py

```diff
@@ -212,3 +247,46 @@ async def test_add_merge_request_labels_invalid_url():
         )
 
     assert "Could not parse merge request URL" in str(exc_info.value)
+
+
+@pytest.mark.asyncio
+async def test_clean_old_repositories(test_directories):
```

Comment:
Sorry, I meant moving all tests and fixtures from here, since the function being tested is now in `utils.py`.